### PR TITLE
gitlab

### PR DESCRIPTION
**Pull request by schottj**
## Description & motivation
Gitlab doc

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 1]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
 ## Overview
 
 Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
```
Comment (dataders) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: nit: but this comes off as a lil' cheesy to me
```suggestion
Once your GitLab account is connected to dbt Cloud, you can:
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 2]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
-- [Trigger CI builds](cloud-enabling-continuous-integration-with-github) when Merge Requests are opened in GitLab.
-- Import new repos with one click
-- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
-## For Developer and Team tiers
+- Log into dbt Cloud via OAuth through GitLab.
```
Comment (dataders) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: is this only for Enterprise tier?

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 3]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
+- Import new GitLab repos with a couple clicks during dbt Cloud project setup.
+- Clone repos using HTTPS rather than SSH.
+- Trigger [Continuous integration](cloud-enabling-continuous-integration-with-github) builds when merge requests are opened in GitLab.
```
Comment (dataders) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md:13]: is there a reason this user permissions bullet got dropped?
```suggestion
- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
- Trigger [Continuous integration](cloud-enabling-continuous-integration-with-github) builds when merge requests are opened in GitLab.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 4]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
 
-To connect your personal GitLab account, navigate to your [Profile](https://cloud.getdbt.com/#/profile/) and select [Integrations](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. 
+The steps to integrate GitLab in dbt Cloud depends on your plan. 
+If you are on:
+- the Developer or Team plan, read these [instructions](/connecting-gitlab#for-dbt-cloud-developer-and-team-tiers).
+- the Enterprise plan, jump ahead to these [instructions](/connecting-gitlab#for-the-dbt-cloud-enterprise-tier).
 
-If your account is not connected, click the button `Link your GitLab account` to continue the setup process. 
-<Lightbox src="/img/docs/dbt-cloud/connecting-gitlab/not-connected.png" title="Link your GitLab" />
+## For dbt Cloud Developer and Team tiers
```
Comment (dataders) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: @runleonarun just flagging a potential usecase for `<tier/>` blocks similar to `<version/>` blocks. not sure how often this comes up though

**Review by dataders — APPROVED** [review]
looks good to me, I just had a few clarifying questions and nits

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 4 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: @dataders I have been thinking of how to handle this situation in the docs UI (a second drop down? a ), but I think you are spot on that we'll need to deal with this use case.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 1 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: @dataders  and @schottj how does this sound?
```suggestion
Connecting your GitLab account to dbt Cloud provides convenience and another layer of security to dbt Cloud:
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 2 above)
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]:
```suggestion
- Log into dbt Cloud using OAuth through GitLab.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 3 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md:13]: @dataders, I think the OAuth bullet point replaces this one.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 5]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
-- Import new repos with one click
-- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
-## For Developer and Team tiers
+- Log into dbt Cloud via OAuth through GitLab.
+- Import new GitLab repos with a couple clicks during dbt Cloud project setup.
+- Clone repos using HTTPS rather than SSH.
+- Trigger [Continuous integration](cloud-enabling-continuous-integration-with-github) builds when merge requests are opened in GitLab.
 
-To connect your personal GitLab account, navigate to your [Profile](https://cloud.getdbt.com/#/profile/) and select [Integrations](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. 
+The steps to integrate GitLab in dbt Cloud depends on your plan.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]:
```suggestion
The steps to integrate GitLab in dbt Cloud depend on your plan:
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 6]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
-- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
-## For Developer and Team tiers
+- Log into dbt Cloud via OAuth through GitLab.
+- Import new GitLab repos with a couple clicks during dbt Cloud project setup.
+- Clone repos using HTTPS rather than SSH.
+- Trigger [Continuous integration](cloud-enabling-continuous-integration-with-github) builds when merge requests are opened in GitLab.
 
-To connect your personal GitLab account, navigate to your [Profile](https://cloud.getdbt.com/#/profile/) and select [Integrations](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. 
+The steps to integrate GitLab in dbt Cloud depends on your plan. 
+If you are on:
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: I think we can remove if we add a colon to previous sentence.
```suggestion
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 7]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
+- the Developer or Team plan, read these [instructions](/connecting-gitlab#for-dbt-cloud-developer-and-team-tiers).
+- the Enterprise plan, jump ahead to these [instructions](/connecting-gitlab#for-the-dbt-cloud-enterprise-tier).
 
-If your account is not connected, click the button `Link your GitLab account` to continue the setup process. 
-<Lightbox src="/img/docs/dbt-cloud/connecting-gitlab/not-connected.png" title="Link your GitLab" />
+## For dbt Cloud Developer and Team tiers
 
-You should be redirected to GitLab and prompted to sign into your account. GitLab will then ask for your explicit authorization: 
+To connect your GitLab account, navigate to **Your Profile** settings, and select **Linked Accounts** from the left navigation pane. There you'll see a **link** to connect your GitLab account.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]:
```suggestion
To connect your GitLab account: 
1. Navigate to Your Profile settings by clicking the gear icon in the top right. 
2. Select **Linked Accounts** in the left menu. 
3. Click **Link** to the right of your GitLab account.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 8]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
 <Lightbox src="/img/docs/dbt-cloud/connecting-gitlab/GitLab-Auth.png" title="GitLab Authorization Screen" />
 
-And you're all set!
+Once you've accepted, you should be redirected back to dbt Cloud, and your integration is ready for developers on your team to [personally authenticate with](/connecting-gitlab#personally-authenticating-with-gitlab).
+
+### Personally authenticating with GitLab
+dbt Cloud developers on the Enterprise plan must each connect their GitLab profiles to dbt Cloud, as every developer's read / write access for the dbt repo is checked in the dbt Cloud IDE.
 
-Non-admins on Enterprise accounts can authenticate by going to `Profile` and then `Integrations` and clicking the `Link your GitLab account` button.
+To connect your personal GitLab account, dbt Cloud developers should navigate to their settings and scroll to **Linked Accounts** under **Your Profile**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]:
```suggestion
To connect a personal GitLab account, dbt Cloud developers should navigate to Your Profile settings by clicking the gear icon in the top right, then select **Linked Accounts** in the left menu.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 9]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
 
-<Lightbox src="/img/docs/dbt-cloud/connecting-gitlab/not-connected.png" title="Link your GitLab" />
+If your GitLab account is not connected, you’ll see "No connected account". Select **Link** to begin the setup process. You’ll be redirected to GitLab, and asked to authorize dbt Cloud in a grant screen.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md:104]:
```suggestion
If your GitLab account is not connected, you’ll see "No connected account." Select **Link** to begin the setup process. You’ll be redirected to GitLab, and asked to authorize dbt Cloud in a grant screen.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md [hunk 10]
```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
...
 
+| Field | Value |
+| ------ | ----- |
+| **Name** | dbt Cloud |
+| **Redirect URI** | https://cloud.getdbt.com/complete/gitlab |
+| **Confidential** | ✔️ |
+| **Scopes** | ✔️ api |
 
-Admins can create a Group Owned Application in GitLab - [GitLab has a guide for that here](https://docs.gitlab.com/ee/integration/oauth_provider.html#group-owned-applications). 
+Note, if you're a dbt Cloud single tenant customer, you'll need to replace **cloud.getdbt.com** with the hostname of
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]:
```suggestion
dbt Cloud single tenant customers need to replace **cloud.getdbt.com** with the hostname of
```

**Review by runleonarun — APPROVED** [review]
@schottj This looks great. Added some minor edits.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 9 above)
Comment (schottj) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md:104]: https://owl.purdue.edu/owl/general_writing/punctuation/quotation_marks/more_quotation_mark_rules.html#:~:text=Place%20the%20punctuation%20outside%20the,%22You%20must%20work%20harder%22%3F

I believe how I have it is correct.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 2 above)
Comment (schottj) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md]: I need to delete this line actually. You can't "log into dbt Cloud via OAuth through GitLab.", we only support that via Github it seems

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md (hunk 3 above)
Comment (schottj) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md:13]: I need to specify "Carry GitLab user permissions through to dbt Cloud IDE's git actions." is only for the enterprise plan. We should delete the "log into dbt Cloud via OAuth through GitLab" altogether. That's incorrect.

**Pull request merged**